### ISSUE TITLE
Plotting accuracy and loss values

### ISSUE THREAD

**Issue by panovr** [facebook/fb.resnet.torch#124]
How can plot loss and accuracy (or top-1/top-5) plots per epoch, for example, using gnuplot?

**Comment by aabobakr** [comment]
I do it with the optim package's logger: this is an example for the training loop in the main.lua file with plotting

```
local errLogger = optim.Logger('error.log')
for epoch = startEpoch, opt.nEpochs do
   -- Train for a single epoch
   local trainTop1, trainTop5, trainLoss = trainer:train(epoch, trainLoader)

   -- Run model on validatioimagenetLabeln set
   local testTop1, testTop5 = trainer:test(epoch, valLoader)
   errLogger:add{['% train top1']    = trainTop1, ['% test top1']    = testTop1}
   local bestModel = false
   if testTop1 < bestTop1 then
      bestModel = true
      bestTop1 = testTop1
      bestTop5 = testTop5
      print(' * Best model ', testTop1, testTop5)
   end
   -- plot logger
   errLogger:style{['% train top1']    = '-', ['% test top1']    = '-'}
   errLogger:plot()
   checkpoints.save(epoch, model, trainer.optimState, bestModel, opt)
end
```

**Comment by panovr** [comment]
Thanks!
